syntax = "proto3";


package users;

// The greeting service definition.
service Users {
  // Sends a greeting
  rpc GetUsers (GetUsersRequest) returns (GetUsersResponse) {}
  rpc CreateUser(CreateUserRequest) returns ( CreateUserResponse) {}
}
message User {
    string id = 1 ;
    string name = 2 ;
    string email = 3 ;
    string password = 4 ;



}


message GetUsersRequest {

}
message GetUsersResponse{
    repeated User users =1 ;

}
message CreateUserRequest{
    User user =1 ;    

}
message CreateUserResponse{
    User user =1;


}